js/ui/date_box/ui.date_box.strategy.list.js

```diff
@@ -44,14 +44,17 @@ var ListStrategy = DateBoxStrategy.inherit({
         return displayFormat || "shorttime";
     },
 
+    isAndroidTheme: function() {
```

Comment:
I saw that there is `isMaterial` function of `themes.js`. I think it would be better to extract `isAndroid5` function to `themes.js` (and add other theme matchers in future, maybe extract them to some kind of `themeUtils` module)